src/contracts/atlas/Atlas.sol

```diff
@@ -192,6 +192,7 @@ contract Atlas is Escrow, Factory {
         uint256[] memory _bidsAndIndices = new uint256[](solverOpsLength);
         uint256 _bidAmountFound;
         uint256 _bidsAndIndicesLastIndex = solverOpsLength - 1; // Start from the last index
+        uint256 _gasWaterMark = gasleft();
```

Comment:
should we add a buffer here?

Reply:
I think either buffer or no buffer is fine, one argument for having a buffer would be to charge for the gas costs of calling `_writeOffBidFindGasCost()` itself. That function is mostly just a TLOAD and a TSTORE which shouldn't be a huge cost, but it could be added to a buffer if desired

Reply:
If its just the single `_writeOffBidFindGasCost()` call, I don't think its worth adding a buffer for the maybe ~250 gas. It increases contract size a bit for the new const, and kinda just offsets the `FIXED_GAS_OFFSET` which I think we'll reduce from 100k to 85k.